modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin_{context}"]
+= Running your dynamic plug-in
+
+You can run the plugin using a local development environment. The OpenShift console will run in a container connected to the cluster you have logged into.
+
+.Prerequisites
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have an OpenShift cluster running.
+* You must have link:https://www.docker.com/[Docker] (for MAC) or link:https://podman.io/[Podman] (for RHEL) installed.
+
+.Procedure
+
+. In one terminal window, run:
++
+[source,terminal]
+----
+$ yarn install
+----
+
+. Run:
++
+[source,terminal]
+----
+$ yarn run start
+----
+
+. In the OpenShift CLI, run:
```

Comment:
```suggestion
. In a second terminal window, run:
```

We probably want to restart the numbering here since these commands are run in a different window.